<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tanulás</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	background-color: rgb(70, 70, 100);
    border: 2px solid #8f8f91;
    border-radius: 10px;
    color: white;
    font-size: 16px;
    font-weight: bold;
}</string>
    </property>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>540</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Vissza</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>540</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Következő</string>
     </property>
    </widget>
    <widget class="QLabel" name="titleLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>911</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	font-size: 30px;
	color: whitesmoke;
}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="contentLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>911</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	border: none;
	color: whitesmoke;
}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>540</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: rgb(70, 70, 100);
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Előző</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
